external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -50,15 +50,7 @@ A column is assumed to be a dimension unless otherwise configured using the `typ
 }
 ```
 
-**If a column mapping is not defined in the `qube-config.json` file for a given CSV column, the column is [configured by convention](./convention.md).**  To ignore a column and to not configure it by convention, the column's definition to `false`. This will ensure the column will not be present in the CSV-W when built by csvcubed.
-
-```json
-{ ...
- "columns": {
-  "Suppressed column": false
```

Comment:
Noooo, this needs to come back. @canwaf added it a week or so again.

Reply:
Will change it back, this must have somehow happened during the merge with main?